exonum-java-binding-common/src/main/proto/block.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+option java_package = "com.exonum.binding.common.serialization";
+option java_outer_classname = "BlockProtos";
```

Comment:
If we remove `java_outer_classname`, it still will be generated by converting the file name to camel case.

Reply:
Not necessarily if `option java_multiple_files = true;`:  https://developers.google.com/protocol-buffers/docs/proto3#options

Reply:
Thanks!
But if we will use the same proto messages as in core (unless they rename those), then we'd have same names for proto messages (`Block`) and our classes (`Block`), so I'd propose to leave outer class. Or we could move all proto messages into one class and have something like `Protos.Block`?

Reply:
If I understand correctly, there are a couple of approaches:
1. Name the proto messages as such, e.g., `BlockProto`
2. Add an outer class for each
3. Add a single outer class for all (i.e., define the messages in a single `.proto` file).

Both (2) and (3) give you an opportunity to `import` a simple name. With (1) you are stuck with `Proto` suffix, but, in cases when you need to use both a message and an application class, that suffix might be more concise than a qualified name (e.g., `BlockProto` vs `BlockProto.Block` vs `CoreProtos.Block`).

Actually, any seems fine to me. If we need to use both proto messages and our classes in the same context, I'd strive to make the code concise.

If there aren't any apparent drawbacks, I'd suggest making (3) change in core. However, it will not be desirable if someone needs only a subset of these proto messages (e.g,. a `Block`, but not a dozen of other proto messages).